File: Module1_AccountName/yeu_cau_he_thong/yeu_cau_6/Customer.js
class Customer {
    constructor() {
        this.name="";
        this.cmnd="";
        this.birthday="";
        this.email="";
        this.address="";
        this.quantityIncluded="";
        this.typeService="";
        this.typeRoom="";
        this.typeCustomer="";
        this.rentDays="";
        this.discount="";
    }
    setName(name) {
        this.name = name;
    };
    getName() {
        return this.name;
    };
    setCMND(cmnd) {
        this.cmnd = cmnd;
    };
    getCMND() {
        return this.cmnd;
    };
    setBirthday(birthday) {
        this.birthday = birthday;
    };
    getBirthday() {
        return this.birthday;
    };
    setEmail(email) {
        this.email = email;
    };
    getEmail() {
        return this.email;
    };
    setAddress(address) {
        this.address = address;
    };
    getAddress() {
        return this.address;
    };
    setTypeCustomer(typeCustomer) {
        this.typeCustomer=typeCustomer;
    };
    getTypeCustomer() {
        return this.typeCustomer;
    };
    setDiscount(discount) {
        this.discount=discount;
    };
    getDiscount() {
        return this.discount;
    };
    setQuantityIncluded(quantityIncluded) {
        this.quantityIncluded=quantityIncluded;
    };
    getQuantityIncluded() {
        return this.quantityIncluded;
    };
    setTypeRoom(typeRoom) {
        this.typeRoom=typeRoom;
    };
    getTypeRoom() {
        return this.typeRoom;
    };
    setRentDays(rentDays) {
        this.rentDays=rentDays;
    };
    getRentDays() {
        return this.rentDays;
    };
    setTypeService(typeService) {
        this.typeService=typeService;
    };
    getTypeService() {
        return this.typeService;
    };
    totalPays() {
        let money=0;
        if(this.getTypeService()==="Villa"){
            money=500;
        }else if (this.getTypeService()==="House"){
            money=300;
        }else if(this.getTypeService()==="Room"){
            money=200;
        }
        return this.getRentDays()*money*(1-this.getDiscount()/100);
    }
}

File: Module3_WBD/10_jsp_jspl/bai_tap/customerlist/src/Customer/Customer.java
package Customer;

public class Customer {
    private String name;
    private String dayOfBirth;
    private String address;
    private String url;

    public Customer() {
    }

    public Customer(String name, String dayOfBirth, String address, String url) {
        this.name = name;
        this.dayOfBirth = dayOfBirth;
        this.address = address;
        this.url = url;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDayOfBirth() {
        return dayOfBirth;
    }

    public void setDayOfBirth(String dayOfBirth) {
        this.dayOfBirth = dayOfBirth;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}
